api/swagger-spec/v1.json

```diff
@@ -2826,6 +3126,75 @@
       "consumes": [
        "*/*"
       ]
+     },
+     {
+      "type": "unversioned.Status",
+      "method": "DELETE",
+      "summary": "delete collection of Node",
+      "nickname": "deletecollectionNamespacedNode",
```

Comment:
noting so I can see if I am not mistaken, but a node is not namespaced, is this a bug?